game/server/tf/tf_obj_sapper.cpp

```diff
@@ -801,6 +801,7 @@ void CObjectSapper::ApplyRoboSapper( CTFPlayer *pTarget, float flDuration, int n
 	if ( IsValidRoboSapperTarget( pTarget ) )
 	{
 		ApplyRoboSapperEffects( pTarget, flDuration );
+		pPlayer->SpeakConceptIfAllowed( MP_CONCEPT_MVM_SAPPED_ROBOT );
```

Comment:
Should this check if they're stealthed?

Reply:
I think SpeakConceptIfAllowed shouldn't allow speaking while cloaked. Regardless, you can't sap while cloaked anyway so hopefully this isn't an issue? If it is, we can do what I did in #132 and put it in game/server/tf/tf_player.cpp instead.